neoantigenPredictor.wdl

```diff
@@ -221,20 +221,28 @@ task extractHLAs{
           ## limit HLA codes to Field 1 and 2, removing everything after (split on :, then join the first two fields)
           allele1=":".join(fields[2].split(":")[0:2])
           allele2=":".join(fields[5].split(":")[0:2])
-          hlas.append([hla_gene,allele1])
-          hlas.append([hla_gene,allele2])
+          if allele1 != ".":
```

Comment:
maybe better to check for an entry that begins with "HLA-" versus looking for the "."

or both. eg. if allele1 !="." and allele1 startswith("HLA-")
-- pseudocode